src/elements/content-uploader/UploadInput.js

```diff
@@ -36,6 +36,7 @@ const UploadInput = ({
         <label className={inputLabelClass} onKeyDown={onKeyDown} tabIndex={0}>
             {inputLabel}
```

Comment:
Can we select this input using user-facing label text rather than a `data-testid` selector?

Reply:
going to add to an aria-label and getByRole instead